crates/viewer/re_view_graph/src/visualizers/edges.rs

```diff
@@ -52,24 +48,38 @@ impl VisualizerSystem for EdgesVisualizer {
     ) -> Result<Vec<re_renderer::QueueableDrawData>, ViewSystemExecutionError> {
         let timeline_query = LatestAtQuery::new(query.timeline, query.latest_at);
 
+        // TODO(cmc): could we (improve and then) use reflection for this?
+        re_types::static_assert_struct_has_fields!(
+            datatypes::Utf8Pair,
+            first: datatypes::Utf8,
+            second: datatypes::Utf8,
+        );
+        let source: ArchetypeFieldName = "first".into();
+        let target: ArchetypeFieldName = "second".into();
```

Comment:
There are a million possible ways to "improve" this. Whether it's via reflection, or codegen, or whatever else.

For now, I very much like the fact that this is so dumb and so low-tech.

We can make it more complex/automated when the need arises.

Reply:
I need to get rid of the `ArchetypeFieldName` though, that's not needed anymore.